modules/grafana/templates/dashboards/deployment_panels/_error_counts_table.json.erb

```diff
@@ -23,36 +23,39 @@
     "col": 1,
     "desc": true
   },
-  "span": 4,
+  "span": 6,
   "styles": [
     {
       "dateFormat": "YYYY-MM-DD HH:mm:ss",
       "pattern": "Time",
       "type": "date"
     },
     {
-      "colorMode": null,
+      "colorMode": "cell",
       "colors": [
-        "rgba(245, 54, 54, 0.9)",
+        "rgba(50, 172, 45, 0.97)",
         "rgba(237, 129, 40, 0.89)",
-        "rgba(50, 172, 45, 0.97)"
+        "rgba(245, 54, 54, 0.9)"
       ],
-      "decimals": 2,
+      "decimals": 0,
       "pattern": "/.*/",
-      "thresholds": [],
+      "thresholds": [
+        "<%= @error_threshold / 2 %>",
```

Comment:
it should do as int / number = int of course @error_threshold is a string so need to cast it to int first. :( just testing it all locally now